SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorSettings.h

```diff
@@ -7,6 +7,165 @@
 
 #include "SpatialGDKEditorSettings.generated.h"
 
+USTRUCT()
+struct FWorldLaunchSection
+{
+	GENERATED_BODY()
+
+	FWorldLaunchSection()
+		: Dimensions(2000, 2000)
+		, ChunkEdgeLenghtMeters(50)
+		, StreamingQueryInterval(4)
+		, SnapshotWritePeriodSeconds(0)
+	{
+		LegacyFlags.Add(TEXT("bridge_qos_max_timeout"), TEXT("0"));
+		LegacyFlags.Add(TEXT("bridge_soft_handover_enabled"), TEXT("false"));
+		LegacyFlags.Add(TEXT("enable_chunk_interest"), TEXT("false"));
+	}
+
+	UPROPERTY(EditAnywhere, config, meta = (ConfigRestartRequired = false, DisplayName = "Dimensions"))
+	FIntPoint Dimensions;
+
+	UPROPERTY(EditAnywhere, config, meta = (ConfigRestartRequired = false, DisplayName = "Chunk edge length in meters"))
+	int32 ChunkEdgeLenghtMeters;
+
+	UPROPERTY(EditAnywhere, config, meta = (ConfigRestartRequired = false, DisplayName = "Streaming query interval"))
+	int32 StreamingQueryInterval;
+
+	UPROPERTY(EditAnywhere, config, meta = (ConfigRestartRequired = false, DisplayName = "Snapshot write period in seconds"))
+	int32 SnapshotWritePeriodSeconds;
+
+	UPROPERTY(EditAnywhere, config, meta = (ConfigRestartRequired = false, DisplayName = "Legacy flags"))
+	TMap<FString, FString> LegacyFlags;
+
+	UPROPERTY(EditAnywhere, config, meta = (ConfigRestartRequired = false, DisplayName = "Legacy java parameters"))
+	TMap<FString, FString> LegacyJavaParams;
+};
+
+USTRUCT()
+struct FWorkerPermissionsSection
+{
+	GENERATED_BODY()
+
+	FWorkerPermissionsSection()
+		: bAllPermissions(true)
+		, bAllowEntityCreation(true)
+		, bAllowEntityDeletion(true)
+		, bAllowEntityQuery(true)
+		, Components()
+	{
+
+	}
+
+	UPROPERTY(EditAnywhere, config, meta = (ConfigRestartRequired = false, DisplayName = "All"))
+	bool bAllPermissions;
+
+	UPROPERTY(EditAnywhere, config, meta = (EditCondition = "!bAllPermissions", ConfigRestartRequired = false, DisplayName = "Allow entity creation"))
+	bool bAllowEntityCreation;
+
+	UPROPERTY(EditAnywhere, config, meta = (EditCondition = "!bAllPermissions", ConfigRestartRequired = false, DisplayName = "Allow entity deletion"))
+	bool bAllowEntityDeletion;
+
+	UPROPERTY(EditAnywhere, config, meta = (EditCondition = "!bAllPermissions", ConfigRestartRequired = false, DisplayName = "Allow entity query"))
+	bool bAllowEntityQuery;
+
+	UPROPERTY(EditAnywhere, config, meta = (EditCondition = "!bAllPermissions", ConfigRestartRequired = false, DisplayName = "Component queries"))
+	TArray<FString> Components;
+};
+
+USTRUCT()
+struct FLoginRateLimitSection
+{
+	GENERATED_BODY()
+
+	FLoginRateLimitSection()
+		: Duration()
+		, RequestsPerDuration(0)
+	{
+
+	}
+
+	UPROPERTY(EditAnywhere, config, meta = (ConfigRestartRequired = false, DisplayName = "Duration"))
```

Comment:
many of the fields are self-describing but some like Duration may benefit from tooltip text explaining possible values